src/stories/components/Sidebar/styles.scss

```diff
@@ -0,0 +1,78 @@
+@import '../../../scss/variables.scss';
+
+// @file
+// Sidebar
+
+.sidebar {
+    /* The sidebar menu */
+    .sidenav {
+        height: 100%; /* Full-height: remove this if you want "auto" height */
+        width: 160px; /* Set the width of the sidebar */
+        position: fixed; /* Fixed Sidebar (stay in place on scroll) */
+        z-index: 1; /* Stay on top */
+        top: 0; /* Stay at the top */
+        left: 0; /* Stay on the left side of the page */
+        background-color: #fff;
+        overflow-x: hidden; /* Disable horizontal scroll */
+        padding-top: 50px;
+    }
+    /* The main content */
+    .main {
+        margin-left: 210px; /* Same as the width of the sidebar + 50px */
+        padding: 20px 0px;
+    }
```

Comment:
I see what you meant to do, but we do not want to style any of the dummy content here. You're styling the content inside the Sidebar component to create the layout, which is not what we want.

Reply:
Yes, that makes more sense. Sorry I got confused a little bit.